.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: "daily"
```

Comment:
In the other repos we use weekly. Did you chose daily on purpose ?

Reply:
Yes, I have daily on `digitalocean` repository too, which is the default interval. I think weekly is for repositories that have too much dependabot updates.